client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -270,17 +271,10 @@ public void testCreateIndex() throws IOException {
     }
 
     public void testDeleteIndex() {
-        DeleteIndexRequest deleteIndexRequest = new DeleteIndexRequest();
-
-        int numIndices = randomIntBetween(0, 5);
-        String[] indices = new String[numIndices];
-        for (int i = 0; i < numIndices; i++) {
-            indices[i] = "index-" + randomAlphaOfLengthBetween(2, 5);
-        }
-        deleteIndexRequest.indices(indices);
+        String[] indices = IndicesClientIT.randomIndicesNames(0, 5);
```

Comment:
I am not too happy with sharing this method between an integration test and this unit test. Furthermore, I have second thoughts on the randomization of indices in the open close index. Maybe we should rather have randomization only here, as in a unit test it fits better, while in an integration test it can cause noise as there are many more moving parts.